api/routes/organizations.js

```diff
@@ -0,0 +1,11 @@
+var express = require('express');
+var router = express.Router();
+
+const Organization = require("../../db/models/Organization");
+
+router.get("/", async (req, res, next) => {
```

Comment:
who would need to use this end points? seems like a dangerous endpoint cos it exposes all our clients.

Reply:
test case use only